consumer/consumer.go

```diff
@@ -102,20 +100,29 @@ func (consumer Impl) Start() error {
 	consumed := 0
 	for {
 		msg := <-consumer.PartitionConsumer.Messages()
-		log.Printf("Consumed message offset %d\n", msg.Offset)
+		consumer.ProcessMessage(msg)
```

Comment:
Probably you want to check if `ProcessMessage` returned an error

Reply:
fixed, nice catch!